js/Column.js

```diff
@@ -0,0 +1,60 @@
+function Column(id, name) {
+	var self = this;
+	
+	this.id = id;
+	this.name = name || 'No name given';
+	this.element = createColumn();
+
+	function createColumn() {
+		// creatin new node
+		var column = $('<div class="column"></div>');
+		var columnTitle = $('<h2 class="column-title">' + self.name + '</h2>');
+		var columnCardList = $('<ul class="card-list"></ul>');
+		var columnDelete = $('<button class="btn-delete">x</button>');
+		var columnAddCard = $('<button class="column-add-card">Dodaj kartę</button>');
+		
+		// significant relevants to the nodes 
+		columnDelete.click(function() {
+			self.deleteColumn();
+		});
+		
+		columnAddCard.click(function(event) {
+			var cardName = prompt("Enter the name of the card");
+			event.preventDefault();
+			$.ajax({
+        	url: baseUrl + '/card',
+        	method: 'POST',
+        	data: {
+              name: cardName,
+    		  bootcamp_kanban_column_id: self.id
+        	},
+        	success: function() {
```

Comment:
you try to use in success function response but you don't send this resposne to your function. it should be in arguments 
```success: function(response) {}```